core/api/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 # async-graphql = { version = "3.0", features = ["tracing"] }
 beef = "0.5"
 
-jsonrpsee = { version = "0.15", features = ["http-server", "macros", "ws-server"] }
+jsonrpsee = { version = "0.16", features = ["client-core", "macros","server"] }
```

Comment:
I think we're not using the client-core feature

Reply:
done